.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,20 @@
+## Context
+
+AI/gen-ai-hub-sdk-js-backlog#ISSUENUMBER.
```

Comment:
[q] Does this work? The backlog is in the internal repo, I think you need the full URL

Reply:
Seems to work for the Java SDK, where they have a similar setup.

Reply:
I tried this in another PR and it didnt resolve the URL

Reply:
Strange, I don't understand how this works for Java then. Will check with them.

Reply:
I added an autolink now, so now the shorthand works.